test/acceptance/tests/consul-dns/consul_dns_test.go

```diff
@@ -0,0 +1,79 @@
+package consuldns
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/consul-helm/test/acceptance/framework"
+	"github.com/hashicorp/consul-helm/test/acceptance/helpers"
+	"github.com/hashicorp/consul/sdk/testutil/retry"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const podName = "dns-pod"
+
+func TestConsulDNS(t *testing.T) {
```

Comment:
What about testing in secure mode? With ACLs enabled we actually need to create an anonymous policy to ensure DNS works so I do think it's worth testing.

Reply:
Aah. I was unaware of the anonymous policy that would be needed to do the same. I'll add a test to test the secure mode installation in the case. Good catch.

Reply:
That's a good point. I completely forgot about the anonymous policy and recommended that Ashwin doesn't test in a secure mode, thinking only of TLS 🤦‍♀️ . @ashwin-venkatesh would you be OK with adding a test for a secure install? I think you could just convert this test to a table test, providing extra helm values for secure config, but keeping all assertions the same. You could take a look at the [basic test](https://github.com/hashicorp/consul-helm/blob/2cdfe72da3c1a0519546d0aa5dddc6c3bb97fa13/test/acceptance/tests/basic/basic_test.go) for reference; I think this one should be pretty similar.

Reply:
:) done!!